Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+
+# docker build --platform=linux/amd64 -t labelmaker:latest .
+# docker run --platform=linux/amd64 -it labelmaker:latest /bin/bash
+
+# Install core developer dependencies
+RUN apt-get -y -m update && apt-get install -y build-essential wget autoconf git unzip python3 python3-pip texlive-base
```

Comment:
I was able to build and run the container, but it appears that `lualatex` and other tex binaries aren't installed:

```bash
root@de056a77a901:/labelmaker# lualatex
bash: lualatex: command not found
root@de056a77a901:/labelmaker# latex
bash: latex: command not found
```

Reply:
Fixed in new PR. I've tested and all seems fine running the following test command from inside the /labelmaker directory: 

./generate_labels.py \
  --project ProjectX \
  --contact MySurname \
  --date 180929 \
  --sample_list sample_list.txt \
  --num_replicates 1 \
  --separator '_'